css/style.css

```diff
@@ -36,23 +43,22 @@ blockquote, dl, p, ol, ul {
 }
 
 body {
-  background: #D4D9DD url('images/bg03.jpg');
+  background: #FFFFFF;
   position: relative;
 }
 
 body, input, textarea, select {
-  color: #474f51;
+  color: #000000;
   font-size: 16px;
-  font-family: 'Oxygen', sans-serif;
+  font-family: 'Jura', sans-serif;
   line-height: 1.45em;
-  font-weight: 400;
+  font-weight: 600;
```

Comment:
We should remove the global font weight here. Having the body copy bold all the time is kinda "shouty".